bin/dredd

```diff
@@ -12,7 +12,7 @@ var path = require('path');
 
 
 var dir = process.env.COVERAGE_DIR ? 'src' : 'lib';
-var CLI = require('../' + dir + '/cli');
+var CLI = require('../' + dir + '/CLI');
```

Comment:
> Acronyms and initialisms should always be all capitalized, or all lowercased.

Wouldn't this change be going in opposite direction? (although personally I dislike this rule)

Reply:
Funny. So from the [examples](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms) it looks like they understand uppercase as "capitalization", even though Wikipedia [describes it](https://en.wikipedia.org/wiki/Capitalization) in the sense in which you understood it. The reasoning,

> Why? Names are for readability, not to appease a computer algorithm.

also supports the idea that they mean uppercase. I think I will send a PR 😄

Reply:
https://github.com/airbnb/javascript/pull/1972